### ISSUE TITLE
[NEED'S LARISSA'S REVIEW/DECISION] Nursing external - pdf - Duplicate locators in index

### ISSUE THREAD

**Issue by caylaeagon** [openstax/cnx-recipes#5411]
# Context
Indexed terms are coded individually in the CNXML. This results in terms that appear more than once per page having more than one locator (page number reference) to that page in the index. These cannot easily be addressed when marking up content because it can't be known how the content will be paginated until it is generated as a PDF.
![Screenshot 2024-01-18 at 10 23 28 AM](https://github.com/openstax/cnx-recipes/assets/99201272/d7d5bcab-f8ed-4fa4-80cf-73b9dc400680)

# New specifications or link to documentation cards
Is it possible to strip out duplicate locators?

## Impact 
Not sure, but I also found it in Introduction to Anthropology.

# Acceptance Criteria
There should be no duplicate locators 

# Content
## Sample Mark-Up
```
<!-- Please paste here if applicable. -->
```

## Location(s)
- **osbooks repo:** https://github.com/openstax/osbooks-nursing-external-bundle
- **collection slug:** All
- **ref/branch:** main
- **chapter/section:** Index
- **surrounding text:**

**Comment by wilkus27** [comment]
There is no an easy way to fix this using css. There are some workarounds using java script. Here are some disscussions:
- https://www.princexml.com/forum/topic/3344/duplicated-page-numbers-in-index
- https://www.princexml.com/forum/topic/1756/index-generation-using-princexml-8 

But I'm wondering whether we really want to fix this. Links are named the same, but they are different elements, that is they're moving us to different places on page:

First link:
![image](https://github.com/openstax/cnx-recipes/assets/40228854/6164b4aa-5ef5-4dd7-89d2-fbc6299a8b48)


Second link:
![image](https://github.com/openstax/cnx-recipes/assets/40228854/2f93fd87-cc8e-4308-8ad0-c74e8eec0518)

Beside that, repetition appears on webview as well:

![image](https://github.com/openstax/cnx-recipes/assets/40228854/8328219a-b26b-43b0-9980-3a48223eaa0c)

This could be easily fixed in cookbook, but from the reader point of view, removing second link has the same effect as just removing second term from the content.

So to sum up, I think we can solve this problem in two ways.

1. Remove from the content terms that appear more then once per module.
2. Rename links, so user know these are two different elements, not a bug. For example `99-1, 99-2`.

**Comment by caylaeagon** [comment]
Thanks, Natalia! I'm hesitant to fix this in the content because it's either a guessing game on how the terms will be paginated or it requires going back into the content and editing out the duplicates after input has been completed.

The webview duplicates make more sense because they take you to the exact location of the term. But, traditionally, print publishing only lists a locator once regardless of how many times the term appears on that page. I'd like to get @openstaxalina's perspective on whether (and how) we should address this or leave it be.

**Comment by openstaxalina** [comment]
Thanks so much to you both. I agree that we do **not** want to fix this in content. 

I thought we had some step in baking that removes duplicate page number listings in the Index, but I'm not finding any evidence of that in GitHub issues. If we don't have this functionality in any of the themes/recipes, then I think we can close this card with Larissa's blessing once she is back. I'll re-assign to Larissa now.

**Comment by lwtchu** [comment]
Agree with folks above. Closing this issue.
